spec/models/location_spec.rb

```diff
@@ -18,4 +18,13 @@
     described_class.create!(location_attrs)
     expect { described_class.create!(location_attrs) }.to raise_error(ActiveRecord::RecordInvalid)
   end
+
+  it "validates size of state abbreviation" do
+    location_attrs = { address: "123 Fake St.",
+                       city: "Bannock",
+                       state: "Idaho",
```

Comment:
Oh, right, sorry, took me waay longer than I'd care to admit to read that as `to_not` instead of `to`... 

FWIW `.to be_present` might read a bit better, but then again I might be a bit biased :)

Also you're assigning to `location_attrs` and then not doing anything with it. My gut is that you meant to do something like `location = described_class.new(location_attrs)` before you run validations?

Also also `location` is [already a method name in rspec](http://www.rubydoc.info/github/rspec/rspec-core/RSpec%2FCore%2FExample%3Alocation), so maybe consider something differently named for the local variable? `loc`? I don't love using abbreviations for things, since grep-ability is a pretty handy thing.